### DISCUSSION TITLE
problema di collegamento con account alexa: notifiche azionabili

### DISCUSSION THREAD

**Discussion by calcioscacchi** [indomus/forum#1740] [HUB: Home Assistant]
Ciaoa  tutti, 

prendendo come riferimento questa guida: https://github.com/keatontaylor/alexa-actions/wiki
e questa discussione: https://spectrum.chat/indomus/general/notifiche-azionabili-di-alexa-una-figata-pazzesca~07b80237-2379-4431-ad06-a0b4b0df6649

ho provato a configurare la skill e mi painto all'account linking. 
L'indirizzo che ho inserito per accedere a HA è quello che uso con duckdns.

Appena provo a collegare l'account, la configurazione sempra a posto: mi viene proposto il collegamento a HA, dove devo semplicemente immettere username e password. 
Dopo qualche istante di riflessione, compare una pagina web di amazon alexa che mi dice 

`Al momento non è possibile collegare la Skill NotificheAzionabili`

Anche seguendo questa nota della guida: 

`> Note: Account linking will not work if your home assistant installation is only reachable by port 8123 (e.g – https://myhainstall.com:8123) If you have home assistant configured this way, please use long lived tokens instead of account linking. Place your long lived token generated in your home assistant install in the double quotes `TOKEN = ""`.`

e inserendo nel file di configurazione il token creato apposta da home assistant, compare lo stesso errore.
Avete dei suggerimenti?
Grazie

**Comment by calcioscacchi** [comment]
Alla fine ho risolto questo errore in questo modo: 

-nel file lambda_function.py:

```
## VERSION 0.8.2

# UPDATE THESE VARIABLES WITH YOUR CONFIG
HOME_ASSISTANT_URL                = 'inserire l'url utilizzato per connettersi ad homeassistant da remoto. Io uso la porta 9000 quindi https://mioHA.duckdns.org:9000'       # REPLACE WITH THE URL FOR YOUR HA FRONTEND
VERIFY_SSL                        = False                           # SET TO FALSE IF YOU DO NOT HAVE VALID CERTS
TOKEN                             = "Inserire il token di lunga durata generato da Home Assistant TRA DOPPI APICI!!"
```


E, al momento di configurare l'account linking, semplicemente spegnerlo visto che usiamo il token.
La skill in questo modo non chiede di collegare l'account ma si collegherà da sola via token. Il test è stato positivo.

**Reply by snakuzzo** [reply]
Scusa, ma questo vuol dire che hai certificati non validi. Non sarebbe meglio lasciare verify ssl a true e sistemare i certificati ?

**Reply by calcioscacchi** [reply]
l'ho lasciato a false per prova, effettivamente non credo che mi comporti dei problemi lasciarlo a true.

**Reply by calcioscacchi** [reply]
@snakuzzo ti confermo che la skill non ha dato problemi nemmeno con SSL a true
